src/main/scala/no/ndla/articleapi/service/converters/HtmlTagGenerator.scala

```diff
@@ -54,11 +54,14 @@ trait HtmlTagGenerator {
       buildEmbedContent(dataAttributes)
     }
 
-    def buildContentLinkEmbedContent(contentId: Long, linkText: String): String = {
-      val dataAttributes = Map(
+    def buildContentLinkEmbedContent(contentId: Long, linkText: String, linkContext: Option[String] = None): String = {
+      val attributes = Map(
         Attributes.DataResource -> ResourceType.ContentLink.toString,
         Attributes.DataContentId -> s"$contentId",
         Attributes.DataLinkText -> linkText)
+
+      val dataAttributes = if(linkContext.isDefined) attributes.updated(Attributes.DataOpenIn, linkContext.toString) else attributes
```

Comment:
Ser veldig bra ut, men her kan vi ikke bruke `..toString`. Det vil resultere i en string "Some(....)". Foreslår noe som dette:
```scala
val dataAttributse = linkContext.map(c => attributes.updated(Attributes.DataOpenIn, c))
.getOrElse(attributes)
```

Reply:
Topp, takk for hjelpen!